### PR TITLE
test: fix argo-ci skipped demo

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/pull/39103#issuecomment-1331627067

### 💡 Background and solution

有一些演示一直没跑到。

截屏2022-12-04 17 49 17

修复后

图片

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   --        |
| 🇨🇳 Chinese |  --         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: tests/shared/demoTest.tsx [hunk 1]
```diff
@@ -56,10 +56,11 @@ function baseText(doInject: boolean, component: string, options: Options = {}) {
   let cssinjsTest = false;
 
   files.forEach((file) => {
-    let testMethod = options.skip === true ? test.skip : test;
-    if (Array.isArray(options.skip) && options.skip.some((c) => file.includes(c))) {
-      testMethod = test.skip;
-    }
```
Comment (afc163) [inline tests/shared/demoTest.tsx:62]: 没有 else 复原，只要跑到一个 skip，后面的也被 skip 掉了。

File: tests/shared/imageTest.tsx [hunk 2]
```diff
@@ -65,14 +65,16 @@ type Options = {
...
-  let testMethod = options.skip === true ? describe.skip : describe;
+  let describeMethod = options.skip === true ? describe.skip : describe;
   const files = glob.sync(`./components/${component}/demo/*.tsx`);
 
   files.forEach((file) => {
     if (Array.isArray(options.skip) && options.skip.some((c) => file.includes(c))) {
-      testMethod = test.skip;
+      describeMethod = describe.skip;
+    } else {
+      describeMethod = describe;
```
Comment (afc163) [inline tests/shared/imageTest.tsx:75]: 没有 else 复原，只要跑到一个 skip，后面的也被 skip 掉了。

**Comment by afc163** [comment]
多了 149 个截图测试用例：

图片

File: components/timeline/__tests__/image.test.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { imageDemoTest } from '../../../tests/shared/imageTest';
 
 describe('Timeline image', () => {
-  imageDemoTest('timeline', { skip: ['pending.tsx'] });
+  imageDemoTest('timeline');
```
Comment (afc163) [inline components/timeline/__tests__/image.test.ts:4]:
```suggestion
  imageDemoTest('timeline', { skip: ['pending.tsx'] });
```

有个 loading 图标，还得 skip 掉。

File: components/steps/__tests__/image.test.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { imageDemoTest } from '../../../tests/shared/imageTest';
 
 describe('Steps image', () => {
-  imageDemoTest('steps', { skip: ['icon.tsx'] });
+  imageDemoTest('steps');
```
Comment (afc163) [inline components/steps/__tests__/image.test.ts:4]:
```suggestion
    imageDemoTest('steps', { skip: ['icon.tsx'] });
```

File: components/steps/__tests__/image.test.ts [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import { imageDemoTest } from '../../../tests/shared/imageTest';
 
 describe('Steps image', () => {
-  imageDemoTest('steps', { skip: ['icon.tsx'] });
+    imageDemoTest('steps', { skip: ['icon.tsx'] });
```
Comment (afc163) [inline components/steps/__tests__/image.test.ts:4]:
```suggestion
  imageDemoTest('steps', { skip: ['icon.tsx'] });
```

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**